devtools/test_dashboard/devtools.js

```diff
@@ -129,7 +129,7 @@ var Tabs = {
         var interval = setInterval(function() {
             if(window.Plotly) {
                 clearInterval(interval);
-                Tabs.setPlotConfig();
+                Tabs.setPlotConfig({ logging: 2 });
```

Comment:
Hmm. This won't work. I think

![image](https://cloud.githubusercontent.com/assets/6675409/15714107/33b69048-27e7-11e6-8311-34d623c0df46.png)

is what you had in mind.